### PR TITLE
Add translation of InstrumentationLibraryInfo to Jaeger

### PR DESCRIPTION
**Pull request by iNikem**
## Changes

Add rules for translating InstrumentationLibraryInfo from OTLP to Jaeger.

### CODE REVIEW

**Comment by tigrannajaryan** [comment]
See my comment https://github.com/open-telemetry/opentelemetry-specification/pull/800#discussion_r470221356

I think it is best to define conventions for all OTLP fields that will be applicable to all other protocols, instead of defining it individually for every protocol (which results in a duplicate like we can see here for Jaeger and in the other PR for Zipkin).

**Comment by bogdandrutu** [comment]
@tigrannajaryan some fields are duplicate indeed, but majority are not. And some protocols needs a specific translation for a field "foo" as attribute in a Span but others may have "Process" like Jaeger.

**Comment by tigrannajaryan** [comment]
> @tigrannajaryan some fields are duplicate indeed, but majority are not. And some protocols needs a specific translation for a field "foo" as attribute in a Span but others may have "Process" like Jaeger.

How about we have a "common" section that is applicable to all protocols, and then individual sections for each protocol that has some specific handling (like Jaeger's Process)?

**Comment by iNikem** [comment]
@tigrannajaryan @bogdandrutu Is it Ok if I create a follow up issue about extracting common translation rules (I may assign it myself) and in the meantime we can merge this and #800 ? I think these two PRs are already useful and make a small step forward. Preparing common document will take more time from me to familiarise myself with protocols.

**Comment by SergeyKanzhelev** [comment]
I will close and re-open PR for auto-assignment

File: specification/trace/sdk_exporters/jaeger.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+This section discusses the details of the transformations between OpenTelemetry
+and Jaeger.
+
+### InstrumentationLibraryInfo
+
+OpenTelemetry Span's `InstrumentationLibraryInfo` MUST be reported as `tags` to Jaeger using the following mapping.
+ 
+| OpenTelemetry | Jaeger |
+| ------------- | ------ |
+| `InstrumentationLibraryInfo.name`|`otel.instrumentationLibrary.name`|
```
Comment (SergeyKanzhelev) [inline specification/trace/sdk_exporters/jaeger.md:25]: Same qq as here: https://github.com/open-telemetry/opentelemetry-specification/pull/800#discussion_r470404555

Comment (iNikem) [reply specification/trace/sdk_exporters/jaeger.md]: Fixed

**Comment by tigrannajaryan** [comment]
> @tigrannajaryan @bogdandrutu Is it Ok if I create a follow up issue about extracting common translation rules (I may assign it myself) and in the meantime we can merge this and #800 ? I think these two PRs are already useful and make a small step forward. Preparing common document will take more time from me to familiarise myself with protocols.

Sounds good. I can also help with that issue.

**Comment by iNikem** [comment]
Created #806 as a follow-up

**Comment by iNikem** [comment]
@SergeyKanzhelev do you still want to block this PR?

**Comment by SergeyKanzhelev** [comment]
swapped issue with @yurishkuro as Yuri might have more context here to merge

**Comment by bogdandrutu** [comment]
@yurishkuro can you confirm is ok? Not too much text but we can have this as a starting point :)

File: specification/trace/sdk_exporters/jaeger.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+and Jaeger.
+
+### InstrumentationLibrary
+
+OpenTelemetry Span's `InstrumentationLibrary` MUST be reported as `tags` to Jaeger using the following mapping.
+ 
+| OpenTelemetry | Jaeger |
+| ------------- | ------ |
+| `InstrumentationLibrary.name`|`otel.instrumentation_library.name`|
+| `InstrumentationLibrary.version`|`otel.instrumentation_library.version`|
```
[the remark this answers is not in the document]
Comment (SergeyKanzhelev) [reply specification/trace/sdk_exporters/jaeger.md:26]: InstrumentationLibrary.name is not an attributes. `InstrumentationLibrary` is a class that Tracer hold on to.

[the remark this answers is not in the document]
Comment (SergeyKanzhelev) [reply specification/trace/sdk_exporters/jaeger.md:26]: see https://github.com/open-telemetry/opentelemetry-specification/pull/800#discussion_r470404555

[the remark this answers is not in the document]
Comment (iNikem) [reply specification/trace/sdk_exporters/jaeger.md:26]: I don't understand what do you mean by "change the naming convention when translating to Jaeger"?

First, as Sergey answered above, `InstrumentationLibrary` is not an attribute, this is a separate field in `InstrumentationLibrarySpans` protocol message.
Second, the same translation, to the same attribute names, was proposed, accepted and merged for OTLP->Zipkin transaltion.

**Review by arminru — APPROVED** [review]

File: specification/trace/sdk_exporters/jaeger.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bogdandrutu) [reply specification/trace/sdk_exporters/jaeger.md:26]: @yurishkuro is this clear now?

**Pull request merged**